CMakeLists.txt

```diff
@@ -98,6 +98,8 @@ if(AV_BUILD_ZLIB)
 set(ZLIB_TARGET zlib)
 ExternalProject_Add(${ZLIB_TARGET}
        URL http://www.zlib.net/zlib-1.2.11.tar.gz
+       URL_HASH MD5=1c9f62f0778697a09d36121ead88e08e
+       DOWNLOAD_DIR ${BUILD_DIR}/download/zlib
```

Comment:
Is there is a reason to not keep the default value?

Reply:
Yes, the default download location is BUILD_DIR, so it's hard to pull out just the downloaded files from other generated content.

Reply:
ok